src/components/common/NumberOfTransactionsWidget/NumberOfTransactionsWidget.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from "prop-types";
+
+export const NumberOfTransactionsWidget = ({
+  entriesOnPage,
+  pageEntriesHandler,
+}) => {
+  return (
+    <>
+      <label htmlFor="transactions-select">
+        Select the number of transactions:
+      </label>
+      <select
+        name="transactions"
+        id="transactions-select"
+        value={entriesOnPage}
+        onChange={(event) =>
+          pageEntriesHandler(Number.parseInt(event.target.value))
+        }
+      >
+        <option value="10">10</option>
```

Comment:
create configuration instead of passing values like that

e.g.:
```
[
   {
      id: 1,
      value: 10,
   },
   ...
    {
      id: 2,
      value: 50,
   },
   ...
}
```

and map through it